core/src/main/java/software/amazon/awssdk/util/StringUtils.java

```diff
@@ -45,10 +44,6 @@
     private static final char CHAR_CARRIAGE_RETURN = '\r';
     private static final char CHAR_FORM_FEED = '\f';
 
-    public static String toString(StringBuilder value) {
```

Comment:
Definitely need this super important method.

Reply:
+1

Reply:
+1. instance methods are for chumps....static all the way